pkg/operator/controller/names.go

```diff
@@ -15,6 +15,12 @@ const (
 	// CA certificate in this namespace.
 	GlobalMachineSpecifiedConfigNamespace = "openshift-config-managed"
 
+	// ClusterScopedNamespace is an empty string representing a cluster scoped namespace.
+	ClusterScopedNamespace = ""
```

Comment:
this just makes it harder to read.

Reply:
This one I agree with.

Reply:
Addressed in the latest version of the PR.